### PR TITLE
Editorial fixes to "vec"

### PR DESCRIPTION
**Pull request by gmlueck**
Fix a few wording problems I noticed when reviewing #479.

### CODE REVIEW

**Review by TApplencourt — APPROVED** [review]

File: adoc/chapters/programming_interface.adoc [hunk 1]
```diff
@@ -17574,14 +17574,14 @@ vec& operator~(const vec& v)
 ----
    a@ Available only when: [code]#DataT != float && DataT != double && DataT != half#.
 
-Construct a new instance of the SYCL [code]#vec# class template with the same template parameters as [code]#v# [code]#vec# with each element of the new SYCL [code]#vec# instance the result of an element-wise [code]#OP# bitwise operation on each element of [code]#v# [code]#vec#.
+Construct a new instance of the SYCL [code]#vec# class template with the same template parameters as [code]#v# [code]#vec# with each element of the new SYCL [code]#vec# instance the result of an element-wise bitwise not operation on each element of [code]#v# [code]#vec#.
```
Comment (keryell) [inline adoc/chapters/programming_interface.adoc:17577]:
```suggestion
Construct a new instance of the SYCL [code]#vec# class template with the same template parameters as [code]#v# [code]#vec# with each element of the new SYCL [code]#vec# instance the result of an element-wise bitwise +not+ operation on each element of [code]#v# [code]#vec#.
```
and everywhere

**Review by keryell — CHANGES_REQUESTED** [review]

File: adoc/chapters/programming_interface.adoc (hunk 1 above)
Comment (keryell) [reply adoc/chapters/programming_interface.adoc:17577]: Actually `[code]#not#`

**Comment by tomdeakin** [comment]
With asciidoc fixes, ready to merge with 2 approvals.

**Review by keryell — APPROVED** [review]
Thanks!

**Pull request merged**